### ISSUE TITLE
Profile picture

### ISSUE THREAD

**Issue by jkbdbrnwsk** [x-team/unleash#456]
Profile picture should be pulled from Google Account.

When clicking on the current picture, user can choose a new avatar - upload it from their computer.

Profile pictures can be changed only by profile owners and Unleashers.

https://cl.ly/1n0t2C302o3f/Image%202017-06-01%20at%206.11.59%20PM.public.png

**Comment by jacekelgda** [comment]
again, not a fan of being in edit mode all the time and changing data just by clicking on it. Should be a separate button/link ( on the picture possibly ) or hover functionality but having in mind mobile version functionality as well.

**Comment by jkbdbrnwsk** [comment]
Yeah, what I actually had in mind is the hover functionality - when an Unleasher or a profile owner hovers that circle, a transparent, greyish overlay with an _edit picture_ text (or something) appears. Clicking on it brings up a standard window for uploading new files.

If we could implement a similar functionality for mobile, would be great, although I don't remember seeing a similar functionality on a phone/tablet - tapping a field and bringing up, say, a Photos library to choose a pic from. I'd leave it to you guys, but I'd put a higher priority on desktop first.
